licenses/licenses.txt

```diff
@@ -0,0 +1,13 @@
+Copyright (c) 2016-present Sonatype, Inc.
```

Comment:
seems to be mixing concerns to do headers along with other changes?
Also this copyright is not in agreement with the header.txt above

Reply:
i had started cobbling together the things being demanded by the license check in the build. I do know now to consistently try to get these to Apache 2 all around. https://jenkins.ci.sonatype.dev/job/integrations/job/cloud/job/Helm3%20Charts/job/Feature%20Snapshot%20Builds/job/INT-4170-publish-helm-repo/

Reply:
i have a separate task to do the license/contributor stuff....would you recommend disabling the license check and waiting to introduce any of this stuff?

Reply:
I'll have to even see if the license check is compatible with the apache license we'll be declaring.

Reply:
OK, I learned better how the license checker works, and I got a good apache 2.0 license into the header.txt, and had the tool re-apply it all around, so it should match better now.